--- a/src/stanExports_lrmconppo.cc
+++ b/src/stanExports_lrmconppo.cc
@@ -7,26 +7,26 @@
 RCPP_MODULE(stan_fit4lrmconppo_mod) {
 
 
-    class_<rstan::stan_fit<stan_model, boost::random::ecuyer1988> >("rstantools_model_lrmconppo")
+    class_<rstan::stan_fit<stan_model, boost::ecuyer1988> >("rstantools_model_lrmconppo")
 
     .constructor<SEXP,SEXP,SEXP>()
 
 
-    .method("call_sampler", &rstan::stan_fit<stan_model, boost::random::ecuyer1988> ::call_sampler)
-    .method("param_names", &rstan::stan_fit<stan_model, boost::random::ecuyer1988> ::param_names)
-    .method("param_names_oi", &rstan::stan_fit<stan_model, boost::random::ecuyer1988> ::param_names_oi)
-    .method("param_fnames_oi", &rstan::stan_fit<stan_model, boost::random::ecuyer1988> ::param_fnames_oi)
-    .method("param_dims", &rstan::stan_fit<stan_model, boost::random::ecuyer1988> ::param_dims)
-    .method("param_dims_oi", &rstan::stan_fit<stan_model, boost::random::ecuyer1988> ::param_dims_oi)
-    .method("update_param_oi", &rstan::stan_fit<stan_model, boost::random::ecuyer1988> ::update_param_oi)
-    .method("param_oi_tidx", &rstan::stan_fit<stan_model, boost::random::ecuyer1988> ::param_oi_tidx)
-    .method("grad_log_prob", &rstan::stan_fit<stan_model, boost::random::ecuyer1988> ::grad_log_prob)
-    .method("log_prob", &rstan::stan_fit<stan_model, boost::random::ecuyer1988> ::log_prob)
-    .method("unconstrain_pars", &rstan::stan_fit<stan_model, boost::random::ecuyer1988> ::unconstrain_pars)
-    .method("constrain_pars", &rstan::stan_fit<stan_model, boost::random::ecuyer1988> ::constrain_pars)
-    .method("num_pars_unconstrained", &rstan::stan_fit<stan_model, boost::random::ecuyer1988> ::num_pars_unconstrained)
-    .method("unconstrained_param_names", &rstan::stan_fit<stan_model, boost::random::ecuyer1988> ::unconstrained_param_names)
-    .method("constrained_param_names", &rstan::stan_fit<stan_model, boost::random::ecuyer1988> ::constrained_param_names)
-    .method("standalone_gqs", &rstan::stan_fit<stan_model, boost::random::ecuyer1988> ::standalone_gqs)
+    .method("call_sampler", &rstan::stan_fit<stan_model, boost::ecuyer1988> ::call_sampler)
+    .method("param_names", &rstan::stan_fit<stan_model, boost::ecuyer1988> ::param_names)
+    .method("param_names_oi", &rstan::stan_fit<stan_model, boost::ecuyer1988> ::param_names_oi)
+    .method("param_fnames_oi", &rstan::stan_fit<stan_model, boost::ecuyer1988> ::param_fnames_oi)
+    .method("param_dims", &rstan::stan_fit<stan_model, boost::ecuyer1988> ::param_dims)
+    .method("param_dims_oi", &rstan::stan_fit<stan_model, boost::ecuyer1988> ::param_dims_oi)
+    .method("update_param_oi", &rstan::stan_fit<stan_model, boost::ecuyer1988> ::update_param_oi)
+    .method("param_oi_tidx", &rstan::stan_fit<stan_model, boost::ecuyer1988> ::param_oi_tidx)
+    .method("grad_log_prob", &rstan::stan_fit<stan_model, boost::ecuyer1988> ::grad_log_prob)
+    .method("log_prob", &rstan::stan_fit<stan_model, boost::ecuyer1988> ::log_prob)
+    .method("unconstrain_pars", &rstan::stan_fit<stan_model, boost::ecuyer1988> ::unconstrain_pars)
+    .method("constrain_pars", &rstan::stan_fit<stan_model, boost::ecuyer1988> ::constrain_pars)
+    .method("num_pars_unconstrained", &rstan::stan_fit<stan_model, boost::ecuyer1988> ::num_pars_unconstrained)
+    .method("unconstrained_param_names", &rstan::stan_fit<stan_model, boost::ecuyer1988> ::unconstrained_param_names)
+    .method("constrained_param_names", &rstan::stan_fit<stan_model, boost::ecuyer1988> ::constrained_param_names)
+    .method("standalone_gqs", &rstan::stan_fit<stan_model, boost::ecuyer1988> ::standalone_gqs)
     ;
 }
--- a/src/stanExports_lrmconppot.h
+++ b/src/stanExports_lrmconppot.h
@@ -671,14 +671,14 @@
       current_statement__ = 38;
       stan::math::validate_non_negative_index("C2", "cn2", cn2);
       current_statement__ = 39;
-      stan::math::validate_non_negative_index("C2", "p", p);
+      stan::math::validate_non_negative_index("C2", "q", q);
       current_statement__ = 40;
       context__.validate_dims("data initialization", "C2", "double",
-        std::vector<size_t>{static_cast<size_t>(cn2), static_cast<size_t>(p)});
-      C2_data__ = Eigen::Matrix<double,-1,-1>::Constant(cn2, p,
+        std::vector<size_t>{static_cast<size_t>(cn2), static_cast<size_t>(q)});
+      C2_data__ = Eigen::Matrix<double,-1,-1>::Constant(cn2, q,
                     std::numeric_limits<double>::quiet_NaN());
       new (&C2) Eigen::Map<Eigen::Matrix<double,-1,-1>>(C2_data__.data(),
-        cn2, p);
+        cn2, q);
       {
         std::vector<local_scalar_t__> C2_flat__;
         current_statement__ = 40;
@@ -686,7 +686,7 @@
         current_statement__ = 40;
         pos__ = 1;
         current_statement__ = 40;
-        for (int sym1__ = 1; sym1__ <= p; ++sym1__) {
+        for (int sym1__ = 1; sym1__ <= q; ++sym1__) {
           current_statement__ = 40;
           for (int sym2__ = 1; sym2__ <= cn2; ++sym2__) {
             current_statement__ = 40;
--- a/src/stanExports_lrmcppo.cc
+++ b/src/stanExports_lrmcppo.cc
@@ -7,26 +7,26 @@
 RCPP_MODULE(stan_fit4lrmcppo_mod) {
 
 
-    class_<rstan::stan_fit<stan_model, boost::random::ecuyer1988> >("rstantools_model_lrmcppo")
+    class_<rstan::stan_fit<stan_model, boost::ecuyer1988> >("rstantools_model_lrmcppo")
 
     .constructor<SEXP,SEXP,SEXP>()
 
 
-    .method("call_sampler", &rstan::stan_fit<stan_model, boost::random::ecuyer1988> ::call_sampler)
-    .method("param_names", &rstan::stan_fit<stan_model, boost::random::ecuyer1988> ::param_names)
-    .method("param_names_oi", &rstan::stan_fit<stan_model, boost::random::ecuyer1988> ::param_names_oi)
-    .method("param_fnames_oi", &rstan::stan_fit<stan_model, boost::random::ecuyer1988> ::param_fnames_oi)
-    .method("param_dims", &rstan::stan_fit<stan_model, boost::random::ecuyer1988> ::param_dims)
-    .method("param_dims_oi", &rstan::stan_fit<stan_model, boost::random::ecuyer1988> ::param_dims_oi)
-    .method("update_param_oi", &rstan::stan_fit<stan_model, boost::random::ecuyer1988> ::update_param_oi)
-    .method("param_oi_tidx", &rstan::stan_fit<stan_model, boost::random::ecuyer1988> ::param_oi_tidx)
-    .method("grad_log_prob", &rstan::stan_fit<stan_model, boost::random::ecuyer1988> ::grad_log_prob)
-    .method("log_prob", &rstan::stan_fit<stan_model, boost::random::ecuyer1988> ::log_prob)
-    .method("unconstrain_pars", &rstan::stan_fit<stan_model, boost::random::ecuyer1988> ::unconstrain_pars)
-    .method("constrain_pars", &rstan::stan_fit<stan_model, boost::random::ecuyer1988> ::constrain_pars)
-    .method("num_pars_unconstrained", &rstan::stan_fit<stan_model, boost::random::ecuyer1988> ::num_pars_unconstrained)
-    .method("unconstrained_param_names", &rstan::stan_fit<stan_model, boost::random::ecuyer1988> ::unconstrained_param_names)
-    .method("constrained_param_names", &rstan::stan_fit<stan_model, boost::random::ecuyer1988> ::constrained_param_names)
-    .method("standalone_gqs", &rstan::stan_fit<stan_model, boost::random::ecuyer1988> ::standalone_gqs)
+    .method("call_sampler", &rstan::stan_fit<stan_model, boost::ecuyer1988> ::call_sampler)
+    .method("param_names", &rstan::stan_fit<stan_model, boost::ecuyer1988> ::param_names)
+    .method("param_names_oi", &rstan::stan_fit<stan_model, boost::ecuyer1988> ::param_names_oi)
+    .method("param_fnames_oi", &rstan::stan_fit<stan_model, boost::ecuyer1988> ::param_fnames_oi)
+    .method("param_dims", &rstan::stan_fit<stan_model, boost::ecuyer1988> ::param_dims)
+    .method("param_dims_oi", &rstan::stan_fit<stan_model, boost::ecuyer1988> ::param_dims_oi)
+    .method("update_param_oi", &rstan::stan_fit<stan_model, boost::ecuyer1988> ::update_param_oi)
+    .method("param_oi_tidx", &rstan::stan_fit<stan_model, boost::ecuyer1988> ::param_oi_tidx)
+    .method("grad_log_prob", &rstan::stan_fit<stan_model, boost::ecuyer1988> ::grad_log_prob)
+    .method("log_prob", &rstan::stan_fit<stan_model, boost::ecuyer1988> ::log_prob)
+    .method("unconstrain_pars", &rstan::stan_fit<stan_model, boost::ecuyer1988> ::unconstrain_pars)
+    .method("constrain_pars", &rstan::stan_fit<stan_model, boost::ecuyer1988> ::constrain_pars)
+    .method("num_pars_unconstrained", &rstan::stan_fit<stan_model, boost::ecuyer1988> ::num_pars_unconstrained)
+    .method("unconstrained_param_names", &rstan::stan_fit<stan_model, boost::ecuyer1988> ::unconstrained_param_names)
+    .method("constrained_param_names", &rstan::stan_fit<stan_model, boost::ecuyer1988> ::constrained_param_names)
+    .method("standalone_gqs", &rstan::stan_fit<stan_model, boost::ecuyer1988> ::standalone_gqs)
     ;
 }
--- a/src/stanExports_lrmconppot.cc
+++ b/src/stanExports_lrmconppot.cc
@@ -7,26 +7,26 @@
 RCPP_MODULE(stan_fit4lrmconppot_mod) {
 
 
-    class_<rstan::stan_fit<stan_model, boost::random::ecuyer1988> >("rstantools_model_lrmconppot")
+    class_<rstan::stan_fit<stan_model, boost::ecuyer1988> >("rstantools_model_lrmconppot")
 
     .constructor<SEXP,SEXP,SEXP>()
 
 
-    .method("call_sampler", &rstan::stan_fit<stan_model, boost::random::ecuyer1988> ::call_sampler)
-    .method("param_names", &rstan::stan_fit<stan_model, boost::random::ecuyer1988> ::param_names)
-    .method("param_names_oi", &rstan::stan_fit<stan_model, boost::random::ecuyer1988> ::param_names_oi)
-    .method("param_fnames_oi", &rstan::stan_fit<stan_model, boost::random::ecuyer1988> ::param_fnames_oi)
-    .method("param_dims", &rstan::stan_fit<stan_model, boost::random::ecuyer1988> ::param_dims)
-    .method("param_dims_oi", &rstan::stan_fit<stan_model, boost::random::ecuyer1988> ::param_dims_oi)
-    .method("update_param_oi", &rstan::stan_fit<stan_model, boost::random::ecuyer1988> ::update_param_oi)
-    .method("param_oi_tidx", &rstan::stan_fit<stan_model, boost::random::ecuyer1988> ::param_oi_tidx)
-    .method("grad_log_prob", &rstan::stan_fit<stan_model, boost::random::ecuyer1988> ::grad_log_prob)
-    .method("log_prob", &rstan::stan_fit<stan_model, boost::random::ecuyer1988> ::log_prob)
-    .method("unconstrain_pars", &rstan::stan_fit<stan_model, boost::random::ecuyer1988> ::unconstrain_pars)
-    .method("constrain_pars", &rstan::stan_fit<stan_model, boost::random::ecuyer1988> ::constrain_pars)
-    .method("num_pars_unconstrained", &rstan::stan_fit<stan_model, boost::random::ecuyer1988> ::num_pars_unconstrained)
-    .method("unconstrained_param_names", &rstan::stan_fit<stan_model, boost::random::ecuyer1988> ::unconstrained_param_names)
-    .method("constrained_param_names", &rstan::stan_fit<stan_model, boost::random::ecuyer1988> ::constrained_param_names)
-    .method("standalone_gqs", &rstan::stan_fit<stan_model, boost::random::ecuyer1988> ::standalone_gqs)
+    .method("call_sampler", &rstan::stan_fit<stan_model, boost::ecuyer1988> ::call_sampler)
+    .method("param_names", &rstan::stan_fit<stan_model, boost::ecuyer1988> ::param_names)
+    .method("param_names_oi", &rstan::stan_fit<stan_model, boost::ecuyer1988> ::param_names_oi)
+    .method("param_fnames_oi", &rstan::stan_fit<stan_model, boost::ecuyer1988> ::param_fnames_oi)
+    .method("param_dims", &rstan::stan_fit<stan_model, boost::ecuyer1988> ::param_dims)
+    .method("param_dims_oi", &rstan::stan_fit<stan_model, boost::ecuyer1988> ::param_dims_oi)
+    .method("update_param_oi", &rstan::stan_fit<stan_model, boost::ecuyer1988> ::update_param_oi)
+    .method("param_oi_tidx", &rstan::stan_fit<stan_model, boost::ecuyer1988> ::param_oi_tidx)
+    .method("grad_log_prob", &rstan::stan_fit<stan_model, boost::ecuyer1988> ::grad_log_prob)
+    .method("log_prob", &rstan::stan_fit<stan_model, boost::ecuyer1988> ::log_prob)
+    .method("unconstrain_pars", &rstan::stan_fit<stan_model, boost::ecuyer1988> ::unconstrain_pars)
+    .method("constrain_pars", &rstan::stan_fit<stan_model, boost::ecuyer1988> ::constrain_pars)
+    .method("num_pars_unconstrained", &rstan::stan_fit<stan_model, boost::ecuyer1988> ::num_pars_unconstrained)
+    .method("unconstrained_param_names", &rstan::stan_fit<stan_model, boost::ecuyer1988> ::unconstrained_param_names)
+    .method("constrained_param_names", &rstan::stan_fit<stan_model, boost::ecuyer1988> ::constrained_param_names)
+    .method("standalone_gqs", &rstan::stan_fit<stan_model, boost::ecuyer1988> ::standalone_gqs)
     ;
 }
